routes/articles.js

```diff
@@ -9,26 +9,45 @@ router.post('/', async (req,res) => {
         title : req.body.title,
         description :  req.body.description,
         author : req.body.author,
-        tags :  req.body.tags
+        tags :  req.body.tags,
+        time : req.body.time
     })
     try {
         const newArticle = await article.save()
         //201 cause its sucess
         res.status(201).json(newArticle)
-    } catch(arr) {
-        res.status(400).json( { message : err.message })
+    } catch(err) {
+        //client's mistake
```

Comment:
see this was the mistake we did manage to find by writing test, Manual testing fails often, Sorry That I didn't pick that up in last PR :)